### PR TITLE
Setup RTR monitoring tab

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
- I cannot track down the ticket but this is the mockup: https://jira.acf.gov/browse/TTAHUB-3493
- Mocked a backend with expected types, handlers, and tests
- Frontend consumes that data

## How to test
Recipient record now has a Monitoring tab 
- Confirm that is only appears to admins or users with feature flag

Visit the monitoring tab
- Confirm that each of the review and citation tabs match the mockup

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-0

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/components/DataCard.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.ttahub-data-card .usa-tag {
+  text-transform: none;
+}
+
+.ttahub-data-card-description-list dd {
+  margin-left: 0;
```
Comment (thewatermethod) [inline frontend/src/components/DataCard.css:6]: can be moved to classname, I think

File: frontend/src/pages/RecipientRecord/pages/Monitoring/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    }
+  }, [currentPage, history, recipientId, regionId]);
+
+  useEffect(() => {
+    async function fetchMonitoringData(slug) {
+      const data = await lookup[slug].fetcher(recipientId, regionId);
+      lookup[slug].setter(data);
+    }
+    if (currentPage && ALLOWED_PAGE_SLUGS.includes(currentPage)) {
+      fetchMonitoringData(currentPage);
```
Comment (thewatermethod) [inline frontend/src/pages/RecipientRecord/pages/Monitoring/index.js:61]: needs try/catch

File: frontend/src/pages/RecipientRecord/pages/Monitoring/components/CitationCard.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.ttahub-monitoring-citation-card .ttahub-monitoring-citation-card-category {
+    min-width: 420px;
```
Comment (thewatermethod) [inline frontend/src/pages/RecipientRecord/pages/Monitoring/components/CitationCard.css:2]: wrap in media query

File: frontend/src/pages/RecipientRecord/pages/Monitoring/components/DescriptionList.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+.ttahub-data-card-description-list > div {
+  flex: 1;
+}
+
+@media (min-width: 1560px) {
+  .ttahub-data-card-description-list {
+    flex-wrap: no-wrap;
+  }
+  .ttahub-data-card-description-list > div {
+    min-width: calc(210px - 2rem);
```
Comment (thewatermethod) [inline frontend/src/pages/RecipientRecord/pages/Monitoring/components/DescriptionList.css:13]: this and the 420px can be css custom props

File: frontend/src/components/TabsNav.js [hunk 5]
```diff
@@ -16,9 +17,17 @@ export default function TabsNav({ backLink, links, ariaLabel }) {
       <nav className="ttahub-tabs-nav_nav bg-white" aria-label={ariaLabel}>
         <ul className="display-flex margin-0 margin-bottom-5 padding-0">
           {links.map((link) => (
-            <li key={uniqueId('tabsnav-link_')} className={liClass}>
-              <NavLink activeClassName={`${linkClass}--active`} className={`${linkClass}`} to={link.to}>{link.label}</NavLink>
-            </li>
+            link.featureFlag ? (
+              <FeatureFlag key={uniqueId('tabsnav-link_')} feature={link.featureFlag}>
```
Comment (nvms) [inline frontend/src/components/TabsNav.js:21]:
```suggestion
              <FeatureFlag key={uniqueId('tabsnav-link_')} flag={link.featureFlag}>
```

File: frontend/src/pages/RecipientRecord/pages/Monitoring/components/SpecialistTags.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '../../../../../components/Tooltip';
+
+export default function SpecialistTags({ specialists }) {
+  return specialists.map((specialist) => {
+    if (!specialist.name) return null;
+
+    return (
+      <p className="usa-prose margin-top-0 margin-bottom-1 margin-right-1 bg-base-lightest radius-md padding-x-1 display-inline-flex flex-align-center flex-justify-between text-decoration-underline">
```
Comment (nvms) [inline frontend/src/pages/RecipientRecord/pages/Monitoring/components/SpecialistTags.js:10]: Seeing a unique key warning on the `p` tag:

```suggestion
import { v4 as uuidv4 } from 'uuid';
import Tooltip from '../../../../../components/Tooltip';

export default function SpecialistTags({ specialists }) {
  return specialists.map((specialist) => {
    if (!specialist.name) return null;

    return (
      <p key={`specialist-${uuidv4()}`} className="usa-prose margin-top-0 margin-bottom-1 margin-right-1 bg-base-lightest radius-md padding-x-1 display-inline-flex flex-align-center flex-justify-between text-decoration-underline">
```

Or whatever makes more sense

**Review by nvms — APPROVED** [review]
I only see two issues here. The key issue isn't that important, but we should fix the feature flag prop name before merging

File: frontend/src/components/TabsNav.js (hunk 5 above)
Comment (thewatermethod) [reply frontend/src/components/TabsNav.js:21]: thanks

File: frontend/src/pages/RecipientRecord/pages/Monitoring/components/SpecialistTags.js (hunk 6 above)
Comment (thewatermethod) [reply frontend/src/pages/RecipientRecord/pages/Monitoring/components/SpecialistTags.js:10]: thanks

**Pull request merged**